doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -53,6 +53,9 @@ the settable parameters that can be used with EVP_PKEY_fromdata().
 I<selection> is described in L</Selections>.
 See L<OSSL_PARAM(3)> for the use of B<OSSL_PARAM> as parameter descriptor.
 
+Parameters in the I<params> array that are not among the settable parameters
```

Comment:
Isn't being in the importable parameters that is important? Settables may be different to importables.

Reply:
The paragraph above explains what settable means in this context.

Reply:
Please note there is a function EVP_PKEY_fromdata_settable(), not importable.

Reply:
> Please note there is a function EVP_PKEY_fromdata_settable(), not importable.

Ugghh. That is really confusing. But ok